uint64 timestamp

uint8 device_address
uint32 device_id
uint32 pcb_id
uint8 firmware_version

uint32 system_uptime
uint32 total_uptime
uint32 total_rope_used

uint8 winch_state
uint8 motor_state
uint8 hook_state
uint8 release_module_state

int16 input_voltage
int16 motor_bus_current
int16 motor_phase_current
int16 release_module_voltage

int16 motor_speed
int16 winch_speed
int16 rope_length
int16 load_weight

int16 fuse_battery_voltage
uint8 fuse_battery_state
uint8 total_fuse_count
uint8 total_overload_count

int16 rope_direction
int16 rope_max_angle
int16 rope_max_angle_reverse
int16 rope_swing_frequency
int16 rope_angular_velocity

uint8 system_healthy
uint8 comm_link_status
